src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/envelopes/DlqReceiverProviderTest.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.domains.envelopes;
+
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.config.QueueConfigurationProperties;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.exceptions.ConnectionException;
+
+class DlqReceiverProviderTest {
+
+    private static final String NAMESPACE = "==namespace==";
+
+    @Test
+    void asd() {
```

Comment:
a bit enigmatic name for a method

Reply:
oh dear. i slipped my quick testing technique out to public. sorry, will fix